magefile.go

```diff
@@ -318,3 +341,25 @@ func findFilesRecursive(match func(path string, info os.FileInfo) bool) ([]strin
 	})
 	return matches, err
 }
+
+// getChangedFiles returns a list of files that have changed in the current git environment
+func getChangedFiles() (map[string]interface{}, error) {
+	// get the list of changed files from git
+	files, err := sh.Output("git", "diff", "--name-only", "HEAD")
```

Comment:
actually..i guess this would fail installing if someone committed changes to `package.json`.. back to the drawing board